.ci/install_opencl.ps1

```diff
@@ -0,0 +1,45 @@
+
+Write-Output "Installing OpenCL CPU platform"
+
+$cache = "$env:PIPELINE_WORKSPACE\opencl_windows-amd_cpu-v3_0_130_135"
+$installer = "AMD-APP-SDKInstaller-v3.0.130.135-GA-windows-F-x64.exe"
+
+if ($env:OPENCL_INSTALLER_FOUND -ne 'true') {
+  # Pipeline cache miss; download OpenCL platform installer executable into workspace cache
+
+  Write-Output "Downloading OpenCL platform installer"
+  Invoke-WebRequest -OutFile "$installer" -Uri "https://github.com/microsoft/LightGBM/releases/download/v2.0.12/$installer"
+
+  Write-Output "Caching OpenCL platform installer"
+  New-Item $cache -ItemType Directory | Out-Null
+  Move-Item -Path "$installer" -Destination "$cache\$installer" | Out-Null
+
+  if (Test-Path "$cache\$installer") {
+    Write-Output "Successfully downloaded OpenCL platform installer"
+  } else {
+    Write-Output "Unable to download OpenCL platform installer"
+    Write-Output "Setting EXIT"
+    $host.SetShouldExit(-1)
+    Exit -1
+  }
+}
+
+# Install OpenCL platform from installer executable expected in workspace cache
+
+Write-Output "Running OpenCL installer"
+Invoke-Command -ScriptBlock {Start-Process "$cache\$installer" -ArgumentList '/S /V"/quiet /norestart /passive /log opencl.log"' -Wait}
```

Comment:
Just a question. What does this syntax mean? `/V" ... opencl.log"`. I mean double quotes.

Reply:
The installer package has a program inside a program: the outer (extractor) program takes the /S and /V arguments.  The value of the /V argument is the list of arguments you want passed to the inner (installer) program.  You need to quote them to make sure they're all passed together, not interpreted as (unrecognized) arguments to the outer program.  Since the ArgumentList is already single-quoted, you have to use double-quotes for the interior quotes.